lib/Site.js

```diff
@@ -295,7 +296,9 @@ Site.prototype.collectUserDefinedVariablesMap = function () {
     const $ = cheerio.load(content);
     $.root().children().each(function () {
       const id = $(this).attr('id');
-      const html = $(this).html();
+      // process the contents of the variable with nunjucks
+      // in case it refers to other variables
```

Comment:
Can this be in one line? Also, it should be sentence case (start with a capital letter).

Reply:
Fixed it